### DISCUSSION TITLE
Markdown Editor: Table without header not rendering in live page

### DISCUSSION THREAD

**Discussion by egitura** [requarks/wiki#6294] [Error / Bug Report] [answered]
When using markdown syntax in the markdown editor to add a table without a header
- the table shows properly in the side by side view of the edit mode
- but the table is not rendered in the live page

Syntax to reproduce:
| - | - |
| test | test | 

I'm currently on the latest (2.5.298) version.

**Comment by phizeroth** [comment, marked as answer]
@egitura I came here to report this as well, but just discovered the solution. There is a setting in `Settings > (Modules) Rendering > MultiMarkdown Table`, make sure that is enabled along with the `Headerless` option. For existing pages you may need to re-save them to re-render.

**Reply by egitura** [reply]
Thank you @phizeroth
